pkg/cmd/server/api/validation/validation.go

```diff
@@ -244,8 +244,8 @@ func ValidateExtendedArguments(config api.ExtendedArguments, flagFunc func(*pfla
 }
 
 func ValidateLDAPSyncConfig(config api.LDAPSyncConfig) ValidationResults {
-	validationResults := ValidateLDAPClientConfig("config",
-		config.Host,
+	validationResults := ValidateLDAPClientConfig("",
```

Comment:
errors will read weird... `".url is invalid..."`
should be using `ValidationResults#Prefix` rather than passing in the parent